scss/_reboot.scss

```diff
@@ -318,15 +316,17 @@ th {
 // Forms
 //
```

Comment:
Left this comment style on purpose? or might be better to change:

```diff
- //
  // Forms
- //
```

Reply:
Woops, fixed it.